lib/beacon/web/live/page_live.ex

```diff
@@ -109,12 +110,15 @@ defmodule Beacon.Web.PageLive do
 
       site ->
         %{"path" => path_info} = params
+
+        # refresh the error handler in case the site changed
+        Beacon.ErrorHandler.enable(site)
```

Comment:
Ah, I think I had an issue with tests failing before, which was fixed by moving `Beacon.ErrorHandler.enable/1` outside the conditional.  But I just tried again, and I think that issue was resolved in another way, because it works fine the way you wrote it.  So I'll move it back inside the conditional